home/home.html

```diff
@@ -1,138 +1,156 @@
+            <img src="./images/works-1.png" alt="dashboard" class="work__image" />
+          </div>
+          <div class="work__content">
+            <h3 class="work__title">Designing Dashboards</h3>
+            <div class="work__info">
+              <span class="work__year">2020</span>
+              <h4 class="work__type">Dashboard</h4>
             </div>
-            <div class="work__content">
-              <h3 class="work__title">36 Days of Malayalam type</h3>
-              <div class="work__info">
-                <span class="work__year">2018</span>
-                <h4 class="work__type">Typography</h4>
-              </div>
-              <p class="work__description">
-                Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation
-                veniam consequat sunt nostrud amet.
-              </p>
+            <p class="work__description">
+              Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis
+              enim velit mollit. Exercitation
+              veniam consequat sunt nostrud amet.
+            </p>
+          </div>
+        </li>
+        <li class="work-list__item">
+          <div class="works__image-wrapper">
+            <img src="./images/works-2.png" alt="vibrant woman portrait" class="work__image" />
+          </div>
+          <div class="work__content">
+            <h3 class="work__title">Vibrant Portraits of 2020</h3>
+            <div class="work__info">
+              <span class="work__year">2018</span>
+              <h4 class="work__type">Illustration</h4>
+            </div>
+            <p class="work__description">
+              Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis
+              enim velit mollit. Exercitation
+              veniam consequat sunt nostrud amet.
+            </p>
+          </div>
+        </li>
+        <li class="work-list__item">
+          <div class="works__image-wrapper">
+            <img src="./images/works-3.png" alt="typography figure" class="work__image" />
+          </div>
+          <div class="work__content">
+            <h3 class="work__title">36 Days of Malayalam type</h3>
+            <div class="work__info">
+              <span class="work__year">2018</span>
+              <h4 class="work__type">Typography</h4>
             </div>
+            <p class="work__description">
+              Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis
+              enim velit mollit. Exercitation
+              veniam consequat sunt nostrud amet.
+            </p>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section class="skills">
+      <div class="skills-graph">
+        <img src="images/skills.svg" alt="skills graph image" class="skills__image">
```

Comment:
Can you make that section html? not an image? For example Progress bars you can use as image. But i think it also can be as  two div containers, and second div can be in % width. If you need some help here ask me I can help.

Reply:
Thank you for review Anton. I am not sure if I can clearly see in Figma design the proportions and distances between all components of this image. Figma shows it as a whole 1 element - image. The only separation I can define there is: 2 containers (images) with language and prof skills and title - so 3 containers. I can't download progress bars separately from that design. Any advice? thanks